client/src/main/java/org/asynchttpclient/netty/channel/ChannelManager.java

```diff
@@ -488,4 +511,11 @@ public ChannelPool getChannelPool() {
     public EventLoopGroup getEventLoopGroup() {
         return eventLoopGroup;
     }
+
+    public ClientStats getClientStats() {
+        final long totalConnectionCount = openChannels.size();
```

Comment:
Hmm, okay. So I guess there's no need to filter out `ServerChannel` instances. Good to know.